BnbnavNetClient/Models/ExtendedLine.cs

```diff
@@ -17,8 +24,12 @@ public double Angle
     {
         get
         {
+            if (Dx == 0)
+            {
+                return Dy < 0 ? 90 : 270;
+            }
             var theta = Math.Atan(-Dy / Dx) * 360.0 / Math.Tau;
-            if (Dx >= 0) theta += 180;
+            if (Dx < 0) theta += 180;
             var thetaNormalized = theta < 0 ? theta + 360 : theta;
```

Comment:
why not use double.Atan2()?
also use double.*** functions now instead of Math.*** (use ``double.Atan2`` and ``double.Tau``)

Reply:
Was unaware of Atan2